tests/regression/02-base/10-init_allfuns.c

```diff
@@ -1,4 +1,4 @@
-// PARAM: --enable allfuns --set ana.activated "['base','threadid','threadflag','escape','mutex','access','mallocWrapper']"
+// PARAM: --enable allfuns --set ana.activated "['base','threadid','threadflag','escape','mutex','access','mallocWrapper']" --disable sem.unknown_function.invalidate.globals
```

Comment:
What's this being added for? To avoid spawning the `pthread_once` stub with `allfuns`? Just add the `goblint_stub` attribute to the stub implementation to achieve that without having to change any configuration.